auslib/web/admin/swagger/api.yaml

```diff
@@ -1597,74 +1963,79 @@ definitions:
 
   DataVersionModel:
     title: Data Version
-    description: version number of object
+    description: "model to re-use 'data_version' field."
     type: object
-    required:
-      - data_version
     properties:
       data_version:
+        description: version number of object
         type: integer
         format: int32
         minimum: 1
         example: 3
 
-  ReleaseBase:
-    title: Release Base
-    description: "To Balrog, a 'release' is data about a related set of builds. This does _not_ match up with the concept of a 'release' being on the 'beta', 'release' or 'esr' channel elsewhere. In Balrog, each set of nightlies on any branch is considered a release"
+  SignoffsRequiredModel:
+    title: Signoffs Required
+    description: "model to re-use 'signoffs_required' field."
+    type: object
+    properties:
+      signoffs_required:
+        description: minimum number of signoffs required to execute the change
```

Comment:
What I'm trying to say is that the signoffs_required is linked with a role, and there may be multiple combinations of role+signoffs_required *before* a change can be executed, so having "to execute the change" here (as part of a single role) is a bit confusing. Perhaps something like "The number of signoffs required for a given role" would be better?